Riot/Modules/Room/ParticipantsInviteModal/ContactsPicker/ContactsPickerViewModel.swift

```diff
@@ -201,12 +201,31 @@ extension ContactsPickerViewModel: ContactsTableViewControllerDelegate {
             return
         }
         
+        // Check for user
+        if MXTools.isMatrixUserIdentifier(contact.displayName) {
+            let user = MXUser(userId: contact.displayName)
+            coordinatorDelegate?.contactsPickerViewModelDidStartValidatingUser(self)
+            user?.update(fromHomeserverOfMatrixSession: self.room.mxSession, success: { [weak self] in
+                guard let self = self else { return }
+                self.coordinatorDelegate?.contactsPickerViewModelDidEndValidatingUser(self)
+                self.displayInvitePrompt(contact: contact)
+            }, failure: { [weak self] error in
+                guard let self = self else { return }
+                self.coordinatorDelegate?.contactsPickerViewModelDidEndValidatingUser(self)
+                self.displayInvitePrompt(contact: contact, contactFound: false)
+            })
+        } else {
+            displayInvitePrompt(contact: contact)
```

Comment:
Just checking with you this branch: 
is it correct calling  `displayInvitePrompt` with `contactFound: true` when `MXTools.isMatrixUserIdentifier` returns false?

Reply:
yes this is correct. FYI This was the previous behaviour before we decided to handle specifically the case where the selected contact is a mxId

Reply:
Ok, it makes sense now! 👍🏻
LGTM, renaming as suggested above would be better.